src/pages/home/report/ReportActionItem.js

```diff
@@ -95,7 +96,9 @@ class ReportActionItem extends Component {
         }
 
         // Only focus the input when user edits a message, skip it for existing drafts being edited of the report.
-        this.textInput.focus();
+        // There is an animation when the comment is hidden and the edit form is shown, and there can be bugs on different mobile platforms
+        // if the input is given focus in the middle of that animation which can prevent the keyboard from opening.
+        focusTextInputAfterAnimation(this.textInput, 10);
```

Comment:
I was testing this at physical android device and I experienced that the edit composer shakes a lit bit sometimes and sometimes the keyboard doesn't open

https://user-images.githubusercontent.com/83179295/207847187-59a9511b-0858-45ea-b0be-dcbe311698db.mp4



I think we can increase the delay. Maybe let's use 100ms same as here https://github.com/Expensify/App/blob/21d9ad71440315d8490eb75c107cad86a82ec2d8/src/pages/home/report/ReportActionCompose.js#L360

At 100ms -

https://user-images.githubusercontent.com/83179295/207847222-7350e688-128e-48b2-bdfb-73cdf3968755.mp4

Reply:
OK, thank you for that testing! I originally had it set to 100, but I was only testing on an emulator (all I have access to) and it worked consistently at 10ms. I'm fine increasing that to 100 👍